### PR TITLE
fix: increase viewbox length of es logo

### PR DESCRIPTION
**Pull request by mpleroux**
### 🔗 Linked issue

[CED-1646: Design System: EnergySage logo gets clipped along right edge](https://energysage.atlassian.net/browse/CED-1646)

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

- Increased viewbox of `ds-es-logo.vue` to allow SVG to scale without being clipped by bounding box
- I didn't see any CSS that set a specific width for the logo

### 🥼 Testing

- Tested navbar and footer locally at http://localhost:8500/organisms/es-nav-bar
- Logo looks fine at any page magnification

#### 🧐 Feedback Requested / Focus Areas

- Are there any downsides to doing this?

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.
- [x] I have documented testing approach

### CODE REVIEW

File: es-design-system/components/ds-es-logo.vue [hunk 1]
```diff
@@ -5,7 +5,7 @@
             height: height,
             width: width,
         }"
-        viewBox="0 0 205 45"
+        viewBox="0 0 215 45"
```
Comment (tomleo) [inline es-design-system/components/ds-es-logo.vue:8]: How did you determine the width should actually be 215px?

**Review by tomleo — APPROVED** [review]
👍

File: es-design-system/components/ds-es-logo.vue (hunk 1 above)
Comment (mpleroux) [reply es-design-system/components/ds-es-logo.vue:8]: I didn't. It's a number that came up in conversation when Eric and I were looking over the problem. 210px also seemed to work.

**Comment by mpleroux** [comment]
I'm unsure about the next steps to take after an `es-ds` PR is approved.

Is the next step to create a release? And should I try it myself following the instructions in the README, or instead send a message to one of the three engineers who do it regularly?

**Pull request merged**